src/main/java/dr/anatolik/controller/Controller.java

```diff
@@ -0,0 +1,39 @@
+package dr.anatolik.controller;
+
+import dr.anatolik.model.Grid;
+import dr.anatolik.view.Printer;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Controller {
+
+    private final Grid grid = new Grid();
+    private final Printer printer = new Printer();
+
+
+    public void startDialog() {
+        System.out.println(printer.printGrid(grid));
+        System.out.println("\nSelect number 0 - 99");
```

Comment:
should prefer to use print vs println when expecting input.

Reply:
Why it is better?

Reply:
normally any cli application asks for a reply on the same line as the question, not on a new line

examples:
https://support.eapps.com/eapps_images/cloud/con_ssh_img/cnnct_ssh_id.png
https://linuxhint.com/wp-content/uploads/2019/10/9-13.png
https://www.howtogeek.com/wp-content/uploads/2016/06/img_5775d608ceb5e.png
https://www.howtogeek.com/wp-content/uploads/2014/03/06_rm_confirmation.png